python/vast/vast/vast.py

```diff
@@ -13,7 +13,7 @@
 
 from .cli import CLI
 
-logger = vast.utils.logging.get("vast.vast")
+logger = vast.utils.logging.get(__name__)
```

Comment:
Same as above :-)

Reply:
`__name__` is vast.vast.vast

Reply:
Ooof, I don't think that's something the user would like.

Reply:
No, I don't know anything better. I'm just surprised that it's repeated *three times*.